Duo/Data/DatabaseConnection.cs

```diff
@@ -11,10 +11,7 @@ public class DataLink
     public DataLink(IConfiguration configuration)
     {
         string? localDataSource = configuration["LocalDataSource"];
-        connectionString = "Data Source=" + localDataSource + ";" +
-                        "Initial Catalog=ISS_Duo;" +
-                        "Integrated Security=True;" +
-                        "TrustServerCertificate=True";
+        connectionString = "Data Source=NANU\\SQLEXPRESS;Initial Catalog=IssDraft;Integrated Security=True;TrustServerCertificate=True;";
```

Comment:
```suggestion
        connectionString = "Data Source=" + localDataSource + ";" +
                         "Initial Catalog=ISS_Duo;" +
                         "Integrated Security=True;" +
                         "TrustServerCertificate=True";
```
dont change this pls, trebuie sa pui nanu etc in appsettings.json